learning_journal/scripts/initializedb.py

```diff
@@ -36,10 +36,10 @@ def main(argv=sys.argv):
     engine = get_engine(settings)
     Base.metadata.create_all(engine)
 
-    session_factory = get_session_factory(engine)
+    # session_factory = get_session_factory(engine)
 
-    with transaction.manager:
-        dbsession = get_tm_session(session_factory, transaction.manager)
+    # with transaction.manager:
+    #     dbsession = get_tm_session(session_factory, transaction.manager)
 
-        model = MyModel(name='one', value=1)
-        dbsession.add(model)
+    #     entry1 = Entry(id='1', title='Day1', body="Some text here.", date="8/24/15")
+    #     dbsession.add(entry1)
```

Comment:
I'd prefer it if you ditched this stuff.  I know it is tempting to keep it around, but remember you have git history to go look for what this file used to look like.  There's really no reason to leave commented code in your files.